feat(curriculum): adding teacher bot workshop (#55870)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/workshop-teacher-chatbot/66b59be6ab830800c4df9146.md

---
id: 66b59be6ab830800c4df9146
title: Step 5
challengeType: 1
dashedName: step-5
---

# --description--

> Now it is time to use the variables you created in the previous step.

Reviewer: ```suggestion
Now, it's time to use the variables you created in the previous step.
```
